### DISCUSSION TITLE
Why the id session length of 22 ? Can we increase it ?

### DISCUSSION THREAD

**Discussion by ThomasCartier** [maxcountryman/tower-sessions#210] [Q&A] [answered]
Hi,
title is the question, 

Thanks!

**Comment by maxcountryman** [comment]
Please see: https://github.com/maxcountryman/tower-sessions/pull/141

**Reply by ThomasCartier** [reply]
Ok so basically, any value greater than 22 bytes leads to a lack of entropy, enabling a hacker to guess the next session cycle, am I right?

**Reply by ThomasCartier** [reply, marked as answer]
Ah, got it, UUIDv4 generates a 128bits random integer.
That's the why. 

Solved!
